cmd/minikube/cmd/version.go

```diff
@@ -43,13 +48,57 @@ var versionCmd = &cobra.Command{
 			"minikubeVersion": minikubeVersion,
 			"commit":          gitCommitID,
 		}
+
+		if listPackagesVersions {
+			co := mustload.Running(ClusterFlagValue())
+			runner := co.CP.Runner
+			// docker version --format='{{.Client.Version}}'
```

Comment:
This comment is just a copy of commands we run below and provides no additional information

Reply:
done